src/audiomixerboard.cpp

```diff
@@ -398,8 +398,8 @@ void CChannelFader::Reset()
     UpdateGroupIDDependencies();
 }
 
-void CChannelFader::SetFaderLevel ( const double dLevel,
-                                    const bool   bIsGroupUpdate )
+void CChannelFader::SetFaderLevel ( const float dLevel,
```

Comment:
Same as above, please do not make any changes to the types in the audiomixerboard. As I wrote earlier, the double type might be necessary on some places.

Reply:
Fixed.